core/src/main/scalajvm/sttp/client4/DefaultSyncBackend.scala

```diff
@@ -1,16 +1,16 @@
 package sttp.client4
 
 import sttp.client4.httpclient.HttpClientSyncBackend
-import sttp.client4.testing.SyncBackendStub
+import sttp.client4.testing.WebSocketBackendStub
 
 object DefaultSyncBackend {
 
   /** Creates a default synchronous backend with the given `options`, which is currently based on
     * [[HttpClientSyncBackend]].
     */
-  def apply(options: BackendOptions = BackendOptions.Default): SyncBackend =
+  def apply(options: BackendOptions = BackendOptions.Default): WebSocketBackend[Identity] =
```

Comment:
This should return a dedicated subtype, `WebSocketSyncBackend`; that's what https://github.com/softwaremill/sttp/issues/1766 is about

Reply:
(the goal is to expose to the user "clean" signatures, without the `Identity` wrapper)